### PR TITLE
Fix/mob 1945 unified UI incorrect background layer is applied for survey input

### PR DESCRIPTION
**Pull request by DavDo**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-1945

### CODE REVIEW

**Review by andrews-moc — APPROVED** [review]

**Review by Pelkar — APPROVED** [review]
LGTM 👍

File: widgetssdk/src/main/java/com/glia/widgets/survey/SurveyAdapter.kt [hunk 1]
```diff
@@ -4,39 +4,14 @@ import androidx.recyclerview.widget.RecyclerView
...
-import com.glia.widgets.view.unifiedui.theme.survey.SurveyBooleanQuestionTheme
-import com.glia.widgets.view.unifiedui.theme.survey.SurveyInputQuestionTheme
-import com.glia.widgets.view.unifiedui.theme.survey.SurveyScaleQuestionTheme
-import com.glia.widgets.view.unifiedui.theme.survey.SurveySingleQuestionTheme
+import com.glia.widgets.survey.viewholder.*
 import java.util.*
-import java.util.function.Consumer
 
 class SurveyAdapter(private val listener: SurveyAdapterListener) :
     RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```
[the remark this answers is not in the document]
Comment (DavDo) [reply widgetssdk/src/main/java/com/glia/widgets/survey/SurveyAdapter.kt]: fixed 30154870e2d826a66c7f0a9b48ac761e5b0bc701

**Comment by DavDo** [comment]
!squash

**Comment by DavDo** [comment]
!merge

**Pull request merged**